benchmarks/build.gradle.kts

```diff
@@ -0,0 +1,66 @@
+
+import kotlinx.benchmark.gradle.JvmBenchmarkTarget
+import kotlinx.benchmark.gradle.benchmark
+
+@Suppress("DSL_SCOPE_VIOLATION")
+plugins {
+    id(libs.plugins.kotlin.multiplatform.id)
+    alias(libs.plugins.kotlin.benchmark)
+    alias(libs.plugins.kotlin.allopen)
+}
+
+allOpen { // jmh benchmark classes must be open
+    annotation("org.openjdk.jmh.annotations.State")
+}
+kotlin {
+    configureMultiplatform(
+        ext = this,
+        explicitApi = false,
+        wasmWasi = false,
+        android = false,
+        linux = false,
+        iOs = false,
+        macOs = false,
+        watchOs = false,
+        tvOs = false,
+        windows = false,
+        wasmJs = false,
+        jvm = true,
+    )
+}
+tasks.withType<JavaExec>().configureEach {
+    jvmArgs("-Dkotlinx.coroutines.debug=off")
+}
+dependencies {
+    commonMainImplementation(projects.core)
+
+    val fluxo = "0.1-2306082-SNAPSHOT"
+    //noinspection UseTomlInstead
+    commonMainImplementation("io.github.fluxo-kt:fluxo-core:$fluxo")
```

Comment:
Remember thant fluxo is not maintained, so this is the latest usable version. I will remove it in the future, but for now it is fine to leave a snapshot since no stable versions were ever published